module/VuFind/src/VuFind/Content/ObalkyKnihService.php

```diff
@@ -180,6 +180,37 @@ public function getData(array $ids): ?\stdClass
     protected function getFromService(array $ids): ?\stdClass
     {
         $param = 'multi';
+        $query = $this->getServiceQuery($ids);
+
+        $url = $this->getBaseUrl();
+        if ($url === '') {
+            $this->logWarning('All ObalkyKnih servers are down.');
+            return null;
+        }
+        $url .= $this->endpoints['books'] . '?';
+        $url .= http_build_query([$param => json_encode([$query])]);
+        $client = $this->getHttpClient($url);
+        try {
+            $response = $client->send();
+        } catch (\Exception $e) {
+            $this->logError('Unexpected ' . $e::class . ': ' . $e->getMessage());
+            return null;
+        }
+        if ($response->isSuccess()) {
+            $json = json_decode($response->getBody());
+            return empty($json) ? null : $json[0];
+        }
+        return null;
+    }
+
+    /**
+     * Get query params for service
+     *
+     * @param array $ids Record identifiers
+     * @return array
```

Comment:
For some reason, this extra line was missing -- `composer fix` should theoretically have added it. I'm going to go ahead and commit the change to save you the trouble. :-)

```suggestion
     *
     * @return array
```